test/tests/Style.js

```diff
@@ -192,3 +192,27 @@ test('setting Group#fillColor and #strokeColor 2', function() {
     // The second path still has its strokeColor set to red:
     equals(secondPath.strokeColor, new Color('red'), 'secondPath.strokeColor');
 });
+
+test('Color change propagation (#1672)', function(assert) {
+    // We use this trick to take a snapshot of the current canvas content
+    // without any kind of side effect that `item.rasterize()` or other
+    // techniques would have.
+    function getDataURL() {
+        view.update();
+        return view.context.canvas.toDataURL();
+    }
+
+    var item = new Path.Circle({
+        center: view.center,
+        radius: 70,
+        fillColor: 'red'
+    });
+    var imageDataBefore = getDataURL();
+
+    // Change style property and check that change was detected.
+    item.fillColor.hue += 100;
+    var imageDataAfter = getDataURL();
+
+    // Canvas content should change after a change of item.fillColor.
```

Comment:
@lehni the comment is here, did you want me to put it somewhere else ?

Reply:
Oh I didn't see it! I meant the string in the `equals()` call, as that's clearer wha the intention was, should it ever fail again.

Reply:
Oh ok, I will change it then